azurerm/internal/services/web/resource_arm_app_service_plan.go

```diff
@@ -267,8 +267,9 @@ func resourceArmAppServicePlanRead(d *schema.ResourceData, meta interface{}) err
 		appServiceEnvironmentId := ""
 		if props.HostingEnvironmentProfile != nil && props.HostingEnvironmentProfile.ID != nil {
 			appServiceEnvironmentId = *props.HostingEnvironmentProfile.ID
+			d.Set("app_service_environment_id", appServiceEnvironmentId)
 		}
-		d.Set("app_service_environment_id", appServiceEnvironmentId)
+		
```

Comment:
Lint error here - this can be quickly resolved with a `make fmt` from the root of the project.

Reply:
Since this field reads the value back from the Azure API - we want to ensure this always has a value in the state file (as such this is the wrong place to set this)

Instead it looks like the check [here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/f9f006e8cfd97aa915aea6db9ff7ebd1b0e56a67/azurerm/internal/services/web/resource_arm_app_service_plan.go#L181) wants updating to set this if the app service environment is not an empty string, rather than checking it gets retrieved as a string?

Reply:
> Lint error here - this can be quickly resolved with a `make fmt` from the root of the project.

I'm trying. Right now the command is giving errors. (I'm on a windows machine)

Reply:
Can someone please help me out with the linting? I removed the empty line.

Reply:
Tried everything last night, but i'm unable to run the make fmt command on my machine.

Reply:
Used go fmt instead.